### ISSUE TITLE
unable to determine selected conversations in gmail using talkback

### ISSUE THREAD

**Issue by GoogleCodeExporter** [2php/eyes-free#398]
```
What steps will reproduce the problem?
1. select a conversation in gmail.
2. Navigate to another conversation by swiping or explore by touch.
3. Use swiping or explore by touch to find the conversation selected. The user 
has no feedback to find the selected conversation.

What is the expected output? What do you see instead?
I expect talkback to notify me which conversations have been selected.  Instead 
I have no way to determine what content is selected.


What version of the product are you using? On what operating system?
Talkback 3.5.2 Android 4.4.2 gmail version 4.9 (1271612)

Please provide any additional information below.
Until recently, it was possible to tell the number of conversations selected, 
but that info was removed with a gmail update.




```

Original issue reported on code.google.com by `hittsj...@gmail.com` on 1 Oct 2014 at 11:17

**Comment by GoogleCodeExporter** [comment]
```
Hello,

It appears as this has been addressed although not in the way you are 
describing.

I am currently testing with Google Talkback 4.3.1 beta 1, Android 5.0, Gmail 
5.5.101116392.release.

In the Gmail main view there is a list of conversations. On the left hand side 
each conversation has a button which says something to the effect: tap to 
select conversation. As long as the conversation is selected text of this left 
hand sided button changes to the opposite tap to unselect the conversation.

So when using explore by touch, it is now possible to identify selected and 
unselected conversations.

On a side note while navigating this list of conversations on the main Gmail 
view, using keyboard only moving up and down using the arrow keys Google 
Talkback reports nothing. Only the audible tone is played indicating the list 
is scrolling. This might be the Gmail issue I am afraid.
```

Original comment by `pvdee...@gmail.com` on 5 Sep 2015 at 11:32
